tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -62,85 +143,154 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
         node_one = worker_nodes_list[0]
         node_two = worker_nodes_list[1]
 
-        # Create a PVC
-        accessmode = constants.ACCESS_MODE_RWX
-        if self.interface == constants.CEPHBLOCKPOOL:
-            accessmode = constants.ACCESS_MODE_RWO
-        pvc_obj = pvc_factory(
-            interface=self.interface,
-            access_mode=accessmode,
-            status=constants.STATUS_BOUND,
-            size="15",
-        )
-
-        # Create a pod on one node
-        logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_one}")
+        time_measures = []
+        for sample_index in range(1, samples_num + 1):
+            # Create a PVC
+            accessmode = constants.ACCESS_MODE_RWX
+            if self.interface == constants.CEPHBLOCKPOOL:
+                accessmode = constants.ACCESS_MODE_RWO
 
-        helpers.pull_images(constants.PERF_IMAGE)
```

Comment:
This is an important part of the test. We pre pull the images on all nodes so new pod creation on a different node doesn't have to wait for images to be pulled.

Reply:
I've moved it into the test itself so that the total test time will include time of this operation.